app/data/models/holiday.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (sequelize, DataTypes) => {
+  const holiday = sequelize.define('holiday', {
+    date: {
+      type: DataTypes.STRING,
```

Comment:
Why this is string? Should be date

Reply:
because otherwise ,  the model will try to create a full date  ( timestamp ) then create the query with the toString which will always return false

Reply:
you can have date with 00 timestamp.. should be alright. There can be potential formatting issue. As long your comparison is on date, should not matter.